src/App/Fossa/VPS/Scan/RunWiggins.hs

```diff
@@ -50,17 +50,19 @@ generateSpectrometerAOSPNoticeArgs :: Severity -> ApiOpts -> ProjectRevision ->
 generateSpectrometerAOSPNoticeArgs logSeverity ApiOpts{..} ProjectRevision{..} ninjaScanId ninjaInputFiles =
   ["aosp-notice-files"]
       ++ optBool "-debug" (logSeverity == SevDebug)
-      ++ ["-endpoint", render apiOptsUri, "-fossa-api-key", unApiKey apiOptsApiKey]
+      ++ ["-fossa-api-key", unApiKey apiOptsApiKey]
       ++ ["-scan-id", unNinjaScanID ninjaScanId]
       ++ ["-name", projectName]
       ++ ["."]
       ++ (T.pack . toFilePath <$> unNinjaFilePaths ninjaInputFiles)
+      ++ optMaybeText "-endpoint" (render <$> apiOptsUri)
```

Comment:
No this is not guaranteed to be `Just`, if the user never sets the value with a flag or config file it will be `Nothing`. The reason this is changed is explained in the PR notes (If this isn't clear let me know, its a bit of scratch notes):
I had to change how we set the default fossa URI. We used to do it after parsing the command-line options, but this made it difficult to adhere to preferring command line > file > default. These changes do not affect behavior for incorrect URIs as far as I am concerned as we only use the default if a user-specified URI has not been provided. That said... I do think we should be doing this validation before we start analysis. It's a little annoying to get to the end and find this out. This will be a follow-up ticket.

What is happening here is that `apiOpts` was changed to have a `Maybe URI` as opposed to previously having a `URI` where the default was already set. What I changed is that the default URI is set right before upload instead of after command-line parsing. The change on this line makes sure we only set endpoint if the user has manually set it.

Reply:
Does wiggins support the default endpoint, and is that guaranteed (meaning they explicitly support it)?  Can we check that before merging at least?

Reply:
Confirmed that Wiggins supports a default endpoint and that this is guaranteed.